src/sass/modules/_m-form-elements.scss

```diff
@@ -186,3 +186,12 @@ button.form-button-disabled {
     }
   }
 }
+
+// Even up the spacing between multi-line labels and their respective elements
+label {
```

Comment:
Assuming Alex signs off on how this looks, I think we will just want to change this for the same inputs that the uswds styles set a top margin for.